Makefile

```diff
@@ -270,10 +270,15 @@ docker-push:
 openshift-deploy: openshift-undeploy
 	@if ! which envsubst > /dev/null 2>&1; then echo "You are missing 'envsubst'. Please install it and retry. If on MacOS, you can get this by installing the gettext package"; exit 1; fi
 	@echo Deploying to OpenShift project ${NAMESPACE}
-	cat deploy/openshift/kiali-configmap.yaml | VERSION_LABEL=${VERSION_LABEL} JAEGER_URL=${JAEGER_URL} GRAFANA_URL=${GRAFANA_URL} envsubst | ${OC} create -n ${NAMESPACE} -f -
-	cat deploy/openshift/kiali-secrets.yaml | VERSION_LABEL=${VERSION_LABEL} envsubst | ${OC} create -n ${NAMESPACE} -f -
-	cat deploy/openshift/kiali.yaml | IMAGE_NAME=${DOCKER_NAME} IMAGE_VERSION=${DOCKER_VERSION} NAMESPACE=${NAMESPACE} VERSION_LABEL=${VERSION_LABEL} VERBOSE_MODE=${VERBOSE_MODE} IMAGE_PULL_POLICY_TOKEN=${IMAGE_PULL_POLICY_TOKEN} envsubst | ${OC} create -n ${NAMESPACE} -f -
-
+	cat deploy/openshift/configmap.yaml | VERSION_LABEL=${VERSION_LABEL} JAEGER_URL=${JAEGER_URL} GRAFANA_URL=${GRAFANA_URL} envsubst | ${OC} create -n ${NAMESPACE} -f -
+	cat deploy/openshift/secrets.yaml | VERSION_LABEL=${VERSION_LABEL} envsubst | ${OC} create -n ${NAMESPACE} -f -
+	cat deploy/openshift/serviceaccount.yaml | VERSION_LABEL=${VERSION_LABEL} envsubst | ${OC} create -n ${NAMESPACE} -f -
+	cat deploy/openshift/service.yaml | VERSION_LABEL=${VERSION_LABEL} envsubst | ${OC} create -n ${NAMESPACE} -f -
+	cat deploy/openshift/route.yaml | VERSION_LABEL=${VERSION_LABEL} envsubst | ${OC} create -n ${NAMESPACE} -f -
+	cat deploy/openshift/deployment.yaml | IMAGE_NAME=${DOCKER_NAME} IMAGE_VERSION=${DOCKER_VERSION} NAMESPACE=${NAMESPACE} VERSION_LABEL=${VERSION_LABEL} VERBOSE_MODE=${VERBOSE_MODE} IMAGE_PULL_POLICY_TOKEN=${IMAGE_PULL_POLICY_TOKEN} envsubst | ${OC} create -n ${NAMESPACE} -f -
+	cat deploy/openshift/clusterrole.yaml | VERSION_LABEL=${VERSION_LABEL} envsubst | ${OC} create -n ${NAMESPACE} -f -
+	cat deploy/openshift/clusterrolebinding.yaml | VERSION_LABEL=${VERSION_LABEL} NAMESPACE=${NAMESPACE} envsubst | ${OC} create -n ${NAMESPACE} -f -
```

Comment:
@lucasponce The reason for this PR is to split the single file into multiple ones, so that it easier to only apply a changed CRD when it has changed.

The above should nevertheless be changed (if possible) to something like `kubectl apply -f deploy/openshift/` which then processes all .yaml files in the directory. I doubt that we need the substitutions on all CRDs.

Reply:
Split the whole file in several minor ones is fine.
I wonder if to install it, it was better to have a built one, just for transactional reasons (order of the files is important?).
Anyway, I suppose we are following the best practices here.

Reply:
@lucasponce the main point here is maintainability and testability of these files. I followed the order it was defined on the file. Istio, for example, separates the files on installation process (https://github.com/istio/istio/tree/master/install/kubernetes/helm/subcharts/kiali/templates).

Since we are using one command like `make openshift-deployment` it will be one command.
I guess I also can create a script like Istio 

`curl -L https://git.io/getLatestIstio | sh -`

to install latest kiali.

Reply:
Is it possible to do something like this:

```
IMAGE_NAME=${DOCKER_NAME} \
IMAGE_VERSION=${DOCKER_VERSION} \
NAMESPACE=${NAMESPACE} \
VERSION_LABEL=${VERSION_LABEL} \
VERBOSE_MODE=${VERBOSE_MODE} \
IMAGE_PULL_POLICY_TOKEN=${IMAGE_PULL_POLICY_TOKEN} \
JAEGER_URL=${JAEGER_URL} \
GRAFANA_URL=${GRAFANA_URL}  \
envsubst | ${OC} create -n ${NAMESPACE} -f /deploy/openshift
```
(notice one command, passing in the directory that contains all the yaml files)

Or maybe use `/deploy/openshift/*.yaml`

Trying to see if we can limit the number of calls here to help limit the number of things a user has to do to get started (these instructions will go up on kiali.io/gettingstarted).

Reply:
For end-user, I have created an install.sh on d5866be which can be used like istio instructions:

 `curl -L https://git.io/getLatestIstio | sh - ` which redirects to this script https://raw.githubusercontent.com/istio/istio/master/release/downloadIstioCandidate.sh


I propose we change to simple one user command `curl -L https://git.io/getLatestKiali | sh - `. If you want I can change the information displayed (send to /dev/null for example) - maybe change envsubst to something more standard like sed.